CHANGELOG.md

```
@@ -39,6 +39,13 @@ Other guiding principles:
 ## v10.11.20260730 _[unreleased; planned for 2026-07-30]_
 
 ## [v10.11.20260723](https://github.com/pragma-org/amaru/releases/tag/v10.11.20260723)
+## v10.11.20260730 _[unreleased; planned for 2026-07-30]_
```

markdownlint-cli2 (0.23.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)